src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String segLine = "    ____________________________________________________________";
+    private static final String indent = "    ";
+    private static final String logo = indent + " ____        _        \n"
+            + indent + "|  _ \\ _   _| | _____ \n"
+            + indent + "| | | | | | | |/ / _ \\\n"
+            + indent + "| |_| | |_| |   <  __/\n"
+            + indent + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String byePhrase = "Bye. Hope to see you again soon!";
```

Comment:
Should constants follow another capitalization convention?
```suggestion
    private static final String SEG_LINE = "    ____________________________________________________________";
    private static final String INDENT = "    ";
    private static final String LOGO = INDENT+ " ____        _        \n"
            + indent + "|  _ \\ _   _| | _____ \n"
            + indent + "| | | | | | | |/ / _ \\\n"
            + indent + "| |_| | |_| |   <  __/\n"
            + indent + "|____/ \\__,_|_|\\_\\___|\n";
    private static final String BYE_PHRASE = "Bye. Hope to see you again soon!";
```